guides/doc-Creating_a_Standard_Operating_Environment/master.adoc

```diff
@@ -0,0 +1,28 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: soe
+
+= {SOEDocTitle}
+
+// This guide is not ready for stable releases
+ifdef::HideDocumentOnStable[]
+include::common/modules/snip_guide-not-ready.adoc[]
+endif::[]
+ifndef::HideDocumentOnStable[]
```

Comment:
Would it make sense to start developing this guide in a way that it is suitable for stable releases? Looks like you're not requesting cherry picks, so that means we only need to make sure the content is OK even after branching.

Reply:
To be honest, I only copy-pasted all this metadata from another guide without really giving it much thought at this point. The next step for this PR is to discuss it with the rest of the team (e.g. does it even make sense to add the SOE content in this way?). If the team agrees that this effort is worth continuing, I'll think about how and where to start publishing it.

Reply:
Considering the size of the SOE guide, I am concerned that the PRs will become unmanageable. Suggest you start with some kind of 'SOE planning' doc (Google doc or miro board) so that you can move sections around and add comments before you make a final decision on how to organize the guide.

Reply:
@apinnick So far, my approach has been to submit any changes to the existing content in smaller PRs (https://github.com/theforeman/foreman-documentation/pull/2847, https://github.com/theforeman/foreman-documentation/pull/2846, https://github.com/theforeman/foreman-documentation/pull/2861), and use this PR to create new SOE-specific files and only minor additional changes. That should prevent this PR from becoming unamangeable. I hope.

Also, as for other tools to plan this, I've tried them all: google doc, miro, a spreadsheet... Miro can't be shared upstream and a spreadsheet turned out to be a lot less useful than a GH preview.

Reply:
Pity about Miro! Drag-and-drop is a lot easier than cut/paste. :-(

Reply:
I think we should not move away from this PR. It's totally OK to apply major concerns now; and then rework the structure in a follow-up PR and fix other minor issues related to wording/style etc.

In the two Google docs for doc PoCs that I have access to, there has not been any discussion in March at all; so I conclude that folks are more active/comfortable on GitHub.

Reply:
> Pity about Miro! Drag-and-drop is a lot easier than cut/paste. :-(

@apinnick On another upstream project (Ansible, I think?), they have good experience with https://www.mindmeister.com/ for the purpose of defining user stories. That does have drag-and-drop functionality. Working through PRs seems to work quite well without the overhead of additional tools, but I could give mindmeister a try too if this team would appreciate it. Let me know.

Reply:
Sounds interesting! I will check it out. Thanks for the link.